boards/arc/axs/doc/index.rst

```diff
@@ -0,0 +1,206 @@
+.. _axs:
+
+DesignWare(R) ARC(R) AXS Software Development platform
+######################################################
+
+Overview
+********
+
+The DesignWare(R) ARC(R) Software Development Platforms are complete, standalone
+platforms that enable software development, debugging and profiling, which consists
+of a mainboard with a CPU card. Synopsys currently has two Software Development
+Platforms available: AXS101 and AXS103
+
+.. image:: ./axs.jpg
+   :width: 442px
+   :align: center
+   :alt: DesignWare(R) ARC(R) AXS Software Development platform (synopsys.com)
+
+For details about the board, see: `DesignWare ARC Software Development Platforms
+<https://www.synopsys.com/dw/ipdir.php?ds=arc-software-development-platform>`__
+
+
+Hardware
+********
+
+For hardware feature details, refer to :
```

Comment:
This would look better as a list:
```
For hardware feature details, refer to:

- `DesignWare ARC AXC001 CPU Card Datasheet
  <https://www.synopsys.com/dw/doc.php/ds/cc/arc_axc001_cpu_card.pdf>`_
- `DesignWare ARC AXC003 CPU Card Datasheet
  <https://www.synopsys.com/dw/doc.php/ds/cc/arc_axc003_cpu_card.pdf>`_
- `DesignWare ARC AXS101 Software Development Platform Datasheet
  <https://www.synopsys.com/dw/doc.php/ds/cc/arc_axs101_sdp.pdf>`_
- `DesignWare ARC AXS103 Software Development Platform Datasheet
  <https://www.synopsys.com/dw/doc.php/ds/cc/arc_axs103_sdp.pdf>`_
```